lib/ziti-tunnel/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ set (LWIP_INCLUDE_DIRS
 include(${LWIP_DIR}/src/Filelists.cmake)
 
 target_sources(lwipcore PRIVATE ${lwip_sys_srcs} lwip/lwiphooks_ip6.c lwip/lwiphooks_ip4.c lwip/lwip_cloned_fns.c)
+target_compile_definitions(lwipcore PUBLIC CMAKE_C_BYTE_ORDER=${CMAKE_C_BYTE_ORDER})
```

Comment:
I would not name it CMAKE_xxx

LWIP_BYTE_ORDER would be better

Reply:
why not just set BYTE_ORDER directly here?

Reply:
Mainly because the value of the cmake variable is either `BIG_ENDIAN` or `LITTLE_ENDIAN`, so we need to make sure those macros are defined as well but cmake doesn't provide them.